doc/users/whats_new/style_changes.rst

```diff
@@ -0,0 +1,98 @@
+Changes to the default style
+----------------------------
+
+The most important changes in matplotlib 2.0 are the changes to the
+default style.
+
+While it is impossible to select the best default for all cases, these
+are designed to work well in the most common cases.
+
+These changes include:
+
+Colors
+``````
+
+- The default figure background color has changed from grey to white.
+
+- The default cycle of colors to draw lines, markers and other content
+  has been changed.  It is based on the `Vega category10 palette
+  <https://github.com/vega/vega/wiki/Scales#scale-range-literals>`__.
+
+- The default color map used for images and pcolor meshes, etc., has
+  changed from ``jet`` to ``viridis``.
+
+- For markers, scatter plots, bar charts and pie charts, there is no
+  longer a black outline around filled markers by default.
+
+- Grid lines are light grey solid 1pt lines.  They are no longer dashed by
+  default.
+
+- The ''Blues'' colormap has been adjusted to be perceptually uniform.
+  The old ``blues`` colormap is available under the name
+  ``legacy_Blues``.
```

Comment:
I don't think that PR has been merged yet, and I'm not sure I see it in this one.